### PR TITLE
[EPA FLIGHTS]: Download module for GHGRP data

### PR DESCRIPTION
**Pull request by beets**
Also outputs crosswalks with ORIS
Setting this as a draft, but sharing early for collaboration

### CODE REVIEW

File: scripts/us_epa/ghgrp/download.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            all_facilities_df = all_facilities_df.append(df)
+        all_facilities_df = all_facilities_df.set_index('Facility Id')
+        all_facilities_df = all_facilities_df.join(oris_df)
+        all_facilities_df.to_csv(self.csv_path('crosswalk.csv'),
+                                 header=True,
+                                 index_label=GHGRP_ID_COL)
+
+
+if __name__ == '__main__':
+    downloader = Downloader('2019')
```
Comment (pradh) [inline scripts/us_epa/ghgrp/download.py:93]: Should we loop over the spreadsheets from all the years, and produce a unified single crosswalk.csv ?

**Review by pradh — COMMENTED** [review]
Thanks for the change!

File: scripts/us_epa/ghgrp/download.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+                continue
+            df = pd.read_csv(csv_path,
+                             usecols=[GHGRP_ID_COL, 'FRS Id'],
+                             dtype=str)
+            all_facilities_df = all_facilities_df.append(df)
+        all_facilities_df = all_facilities_df.join(
+            oris_df.set_index(GHGRP_ID_COL), on=GHGRP_ID_COL, how='left')
+        all_facilities_df.to_csv(self._csv_path('crosswalk.csv'),
+                                 header=True,
+                                 index=None)
```
Comment (pradh) [inline scripts/us_epa/ghgrp/download.py:132]: do we need this to_csv ?

**Review by pradh — APPROVED** [review]
Nice!

Might add a TODO for README and tests?

File: scripts/us_epa/ghgrp/download.py (hunk 2 above)
Comment (beets) [reply scripts/us_epa/ghgrp/download.py:132]: ah, not anymore. initially i was thinking of processing each year horizontally, but it makes more sense to do it by sheet across years. deleted.

**Comment by beets** [comment]
thanks! added todo as well

**Pull request merged**